app/main/posts/views/mode-context-form-filter.directive.js

```diff
@@ -45,6 +48,13 @@ function ModeContextFormFilter($scope, FormEndpoint, PostEndpoint, $q, _, $rootS
         });
     }
 
+    function externalOpenDropDown() {
```

Comment:
You shouldn't need to manually trigger a click handler. dropdown directive has an `is-open` property. Add that to the dropdown tag and then just set the variable true or false.